### PR TITLE
Hotfix 7.2.11

### PR DESCRIPTION
**Pull request by yvesgoeleven**
Fixes https://github.com/Particular/NServiceBus.AzureServiceBus/issues/686

### CODE REVIEW

**Comment by danielmarbach** [comment]
@yvesgoeleven so no null check required for the equality? Could we have a test that verifies that corner case as well?

**Comment by yvesgoeleven** [comment]
@danielmarbach I'll add a test, but the check uses ReferenceEquals which won't throw nullrefs

File: src/Tests/Topology/MetaModel/When_parsing_string_to_connection_string.cs [hunk 1]
```diff
@@ -233,5 +233,39 @@ public void Should_be_able_to_parse_various_domain_names(string connectionString
...
+            ConnectionString connectionString1, connectionString2;
+            ConnectionString.TryParse(connectionStringString, out connectionString1);
+            ConnectionString.TryParse(connectionStringString, out connectionString2);
+
+            var areEqual = connectionString1.Equals(connectionString2);
+            Assert.IsTrue(areEqual);
+        }
+
+        [Test]
+        [TestCase("Endpoint=sb://namespacename.servicebus.windows.net", "Endpoint=sb://namespacename.servicebus.windows.net;SharedAccessKeyName=name;SharedAccessKey=key")]
```
Comment (danielmarbach) [inline src/Tests/Topology/MetaModel/When_parsing_string_to_connection_string.cs]: I do think using test cases here is a bit of an odd choice but I'm not against it. So approving

Comment (SeanFeldman) [reply src/Tests/Topology/MetaModel/When_parsing_string_to_connection_string.cs]: Tend to agree. When I see `TestCase`, I think of permutations.
Considering that this will be ported forward, could be simplified.
